pennylane_lightning_gpu/src/simulator/CMakeLists.txt

```diff
@@ -1,8 +1,9 @@
 project(lightning_gpu_simulator)
 set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_CUDA_STANDARD 17)
 enable_language(CXX CUDA)
 
-set(SIMULATOR_FILES StateVectorCudaBase.hpp StateVectorCudaManaged.hpp cuGateCache.hpp cuGates_host.hpp CACHE INTERNAL "" FORCE)
+set(SIMULATOR_FILES StateVectorCudaBase.hpp StateVectorCudaManaged.hpp cuGateCache.hpp cuGates_host.hpp init.cu CACHE INTERNAL "" FORCE)
```

Comment:
Due to updates in the overlapping PR, some changes will likely be required here around `init.cu` (we added a similar `initSV.cu` ).

Reply:
this is obsolete.